assets/css/styles.scss

```diff
@@ -664,3 +664,59 @@ body {
     }
   }
 }
+
+// Outreach partner section
+.outreach-grayscale {
+	.row {
+		text-align: center;
+		margin-left: auto;
+		margin-right: auto;
+		margin-bottom: 20px;
+		&:after {
+			content: "";
+			display: table;
+			clear: both;
+		}
+		.col {
+			float: left;
```

Comment:
Again, using flex, would avoid the use of `float`. I would prefer avoiding the use of `float` for laying out things in a row.